src/ralph/back_office/tests/factories.py

```diff
@@ -51,7 +51,7 @@ class BackOfficeAssetFactory(DjangoModelFactory):
     force_depreciation = False
     office_infrastructure = factory.SubFactory(OfficeInfrastructureFactory)
     sn = factory.Faker('ssn')
-    barcode = factory.Faker('ean8')
+    barcode = factory.Sequence(lambda n: str(n + 10**8))
```

Comment:
Shouldn't this be more like this?
`barcode = factory.Sequence(lambda n: str(10**8 + 8))`